caddyconfig/httpcaddyfile/tlsapp.go

```diff
@@ -382,5 +382,9 @@ func consolidateAutomationPolicies(aps []*caddytls.AutomationPolicy) []*caddytls
 		return len(aps[i].Subjects) > len(aps[j].Subjects)
 	})
 
+	for i := 0; i < len(aps); i++ {
+		sort.Strings(aps[i].Subjects)
```

Comment:
Why is this needed?

Reply:
So the diffing approach treats the out puts as text, there are few places that iterate over maps and things which will randomly reorder the outputs

Reply:
Cool, sounds good to me.